test/suiteconfig/16hr.suite.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Host": "./corerun",
```

Comment:
Note: Going forward, be careful here. This string exactly ends up in our executing batch and shell scripts that invoke our tests. Using ./corerun doesn't fly in batch. In the positive side, we will prepend the ./ when producing the shell file, so that we are using the host that sits in the test directory.